### DISCUSSION TITLE
Questions about structures in sys_configs parameter

### DISCUSSION THREAD

**Discussion by PolyuWeldingSpock** [deepmodeling/dpgen#1288] [Q&A] [answered]
Hi everyone,

Recently, I have been trying to run the DPGEN example, and I noticed that there are two folders in the input file param.json, one is the initial data folder "init_data", and the other is "sys_configs". I have read some articles, and it seems that "sys_configs" is used to verify the accuracy of the model trained in the previous step, and to run LAMMPS. I would like to ask whether the structures in "sys_configs" can be different from those in "init_data", for example, if "init_data" has a 2x2 unit cell and "sys_configs" has a 4x4 supercell. Also, I would like to ask how you usually choose the structures in "sys_configs".

Hope someone could solve my problem. Thank you!

**Comment by wanghan-iapcm** [comment, marked as answer]
init_data is initial training data.

sys_configs provides the initial configurations for exploration MD simulations.
